templates/common/_base/files/kubelet-log-level.yaml

```diff
@@ -3,4 +3,4 @@ path: "/etc/systemd/system/kubelet.service.d/kubelog.conf"
 contents:
   inline: |
     [Unit]
-    Environment="KUBELET_LOG_LEVEL=3"
+    Environment="KUBELET_LOG_LEVEL=2"
```

Comment:
i thought level 2 was not sufficient for debugging?

Reply:
Problem is that level 3 is logging dead pod logs into the kubelet log. Instead of chasing all level 3 logs... I'm proposing just setting it to level 2. This new approach with a kubelog.conf would allow us to manage the kubelet log level within the kubeletconfig perhaps.

[Log Line](https://github.com/openshift/origin/blob/release-4.4/vendor/k8s.io/kubernetes/pkg/kubelet/status/status_manager.go#L434)

Reply:
I'm open to changing the log line to be 4, but it seems like whack-a-mole.

![](https://media.giphy.com/media/Jaho05MluSaQM/giphy.gif)

Reply:
If there's consensus that level work works between the experts (ie you and @sjenning) I'm ok with it, just want to make sure this isn't going to hurt us later.

@sjenning wdyt?

Reply:
@kikisdeliveryservice I'm on board with this